aten/src/ATen/core/GeneratorForPrivateuseone.cpp

```diff
@@ -34,7 +34,7 @@ at::Generator GetGeneratorForPrivateuse1(c10::DeviceIndex device_index) {
   TORCH_CHECK(
       GetGeneratorPrivate().has_value(),
       "Please register a generator to the PrivateUse1 dispatch key, \
-      using the REGISTER_GENERATOR_PRIVATEUSE1 macro.");
+      you should derive PrivateUse1HooksInterface to implememt getNewGenerator.")
```

Comment:
If you want to use `GetGeneratorForPrivateuse1`, you also need to use `REGISTER_GENERATOR_PRIVATEUSE1` to register. The two need to be used together.

Reply:
@FFFrog Thanks for the review. I'm gonna close this.